cicecore/cicedynB/general/ice_init.F90

```diff
@@ -1775,7 +1775,7 @@ subroutine input_data
           grid_type  /=  'rectangular'    .and. &
           grid_type  /=  'cpom_grid'      .and. &
           grid_type  /=  'regional'       .and. &
-          grid_type  /=  'latlon' ) then
+          grid_type  /=  'setmask' ) then
```

Comment:
The latlon grid functionality has changed here. We are using a new NUOPC mesh functionality. Are there other groups that use the 'latlon' grid?

Reply:
We need to keep the latlon capability.  F cases use the latlon feature in prescribed ice mode, I assume there may be other users of older version of CESM that need this.  RASM certainly does.

Reply:
I didn't realize that RASM still did F compsets! We can certainly keep this. I can make this modification.